apm-agent-plugins/apm-jakarta-websocket-plugin/src/main/java/co/elastic/apm/agent/websocket/JakartaServerEndpointInstrumentation.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package co.elastic.apm.agent.websocket;
+
+import co.elastic.apm.agent.bci.bytebuddy.SimpleMethodSignatureOffsetMappingFactory;
+import co.elastic.apm.agent.impl.ElasticApmTracer;
+import co.elastic.apm.agent.util.VersionUtils;
+import jakarta.websocket.server.ServerEndpoint;
+import net.bytebuddy.asm.Advice;
+
+import javax.annotation.Nullable;
+import java.util.Collection;
+import java.util.Collections;
+
+public class JakartaServerEndpointInstrumentation extends BaseServerEndpointInstrumentation {
+
+    public JakartaServerEndpointInstrumentation(ElasticApmTracer tracer) {
+        super(tracer);
+    }
+
+    @Override
+    public Collection<String> getInstrumentationGroupNames() {
+        return Collections.singletonList("jakarta-websocket");
```

Comment:
So that you can disable all websocket instrumentations with just `disable_instrumentations=websocket`.
```suggestion
        return Arrays.asList("websocket", "jakarta-websocket");
```
Don't forget the import.